src/autora/state/delta.py

```diff
@@ -206,9 +206,10 @@ def wrap_to_use_state(f):
     It was inspired by the pytest "fixtures" mechanism.
 
     Args:
-        f:
+        f: target function to be wrapped such that it can recieve arguments from and
+           contribute Deltas to `State` objects.
```

Comment:
I'd probably describe this argument in terms of the argument's properties, rather than the intention of the wrapper function.
```suggestion
        f: a function with arguments that could be fields on a `State` 
            and that returns a `Delta`.
```

Reply:
better, thanks!